### ISSUE TITLE
combine standard ddoc-umentation and wiki-style editing on single pages

### ISSUE THREAD

**Issue by dlangBugzillaToGithub** [dlang/dlang.org#3943]
### Description

````markdown
For me, the power of wiki-style editing is pretty convincing. It seems generally agreed that what documentation can be easily edited, should be. Existing in-code documentation requires the review process, which is at odds with the lightning quick way a wiki page is normally changed.

This enhancement proposes an intermediate solution to the problem. Each std function, for example, will have a top (or bottom) window which formats and copies the existing embedded ddoc-umentation, while at the bottom (or top) providing a freestyle wiki window. Editing the embedded docs from the wiki page will still be a little clunky like it is now (with the "improve this page" button), because of github pulls and the review process, but editing the wiki window will harness the power of wiki.

In other words, you shouldn't have to click "Page wiki" to get to the wiki, and no Wiki page should exclude what's already in the ddocs. The documentation and the wiki should be the same page, while working around the existing barriers to accomplishing this.

The transition could be piecemeal, with a wiki window being merged onto the dlang.org page one function at a time, or, conversely, with links to functions with wiki pages somehow overriding the default dlang.org page.

Basically, I want everyone to be on the same page (pun intended).
````
